src/components/Header/index.tsx

```diff
@@ -17,8 +17,8 @@ const Header = (): JSX.Element => {
     if (isKeyPress) {
       return;
     }
-    const target = e.target as HTMLElement;
-    const toggle = target.innerHTML.includes('nights_stay') ? 'dark' : 'light';
+    const target = e.target as any;
+    const toggle = target.innerText.includes('nights_stay') ? 'dark' : 'light';
```

Comment:
It will make more sense if the name of constant will be theme instead of toggle. @benhalverson what do you think about this ?